<ui version="4.0" >
 <author>John Eric Martin &lt;john.eric.martin@gmail.com></author>
 <class>PaletteEditor</class>
 <widget class="QWidget" name="PaletteEditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Palette Editor</string>
  </property>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>360</y>
     <width>551</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title" >
    <string>Colors</string>
   </property>
   <widget class="QWidget" name="gridLayout" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QGridLayout" >
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <property name="horizontalSpacing" >
      <number>6</number>
     </property>
     <property name="verticalSpacing" >
      <number>6</number>
     </property>
     <item row="0" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Group:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Role:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QComboBox" name="RoleCombo" />
     </item>
     <item row="0" column="1" >
      <widget class="QComboBox" name="GroupCombo" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayout_3" >
    <property name="geometry" >
     <rect>
      <x>260</x>
      <y>20</y>
      <width>281</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="ColorButton" />
       </item>
       <item>
        <widget class="QLabel" name="label_7" >
         <property name="text" >
          <string>R:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="RBox" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximum" >
          <number>255</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5" >
         <property name="text" >
          <string>G:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="GBox" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximum" >
          <number>255</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6" >
         <property name="text" >
          <string>B:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="BBox" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximum" >
          <number>255</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="Description" >
       <property name="font" >
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="text" >
        <string>Role Description Goes Here</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="PreviewBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>551</width>
     <height>341</height>
    </rect>
   </property>
   <property name="autoFillBackground" >
    <bool>true</bool>
   </property>
   <property name="title" >
    <string>Preview</string>
   </property>
   <widget class="QWidget" name="horizontalLayout" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>531</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QTreeWidget" name="treeWidget" >
       <property name="alternatingRowColors" >
        <bool>true</bool>
       </property>
       <column>
        <property name="text" >
         <string>Column 1</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Column 2</string>
        </property>
       </column>
       <item>
        <property name="text" >
         <string>Item 1</string>
        </property>
        <property name="text" >
         <string>Item 2</string>
        </property>
        <item>
         <property name="text" >
          <string>Item A</string>
         </property>
         <property name="text" >
          <string>Item B</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Item C</string>
         </property>
         <property name="text" >
          <string>Item D</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Item E</string>
         </property>
         <property name="text" >
          <string>Item F</string>
         </property>
        </item>
       </item>
       <item>
        <property name="text" >
         <string>Item 3</string>
        </property>
        <property name="text" >
         <string>Item 4</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Item 5</string>
        </property>
        <property name="text" >
         <string>Item 6</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Item 7</string>
        </property>
        <property name="text" >
         <string>Item 8</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Item 9</string>
        </property>
        <property name="text" >
         <string>Item 10</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QScrollBar" name="verticalScrollBar" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="currentIndex" >
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab" >
        <attribute name="title" >
         <string>Tab 1</string>
        </attribute>
        <widget class="QWidget" name="verticalLayout" >
         <property name="geometry" >
          <rect>
           <x>10</x>
           <y>10</y>
           <width>221</width>
           <height>276</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="leftMargin" >
           <number>0</number>
          </property>
          <property name="topMargin" >
           <number>0</number>
          </property>
          <property name="rightMargin" >
           <number>0</number>
          </property>
          <property name="bottomMargin" >
           <number>0</number>
          </property>
          <item>
           <layout class="QGridLayout" >
            <property name="leftMargin" >
             <number>0</number>
            </property>
            <property name="topMargin" >
             <number>0</number>
            </property>
            <property name="rightMargin" >
             <number>0</number>
            </property>
            <property name="bottomMargin" >
             <number>0</number>
            </property>
            <property name="horizontalSpacing" >
             <number>6</number>
            </property>
            <property name="verticalSpacing" >
             <number>6</number>
            </property>
            <item row="4" column="1" >
             <widget class="QCheckBox" name="checkBox_2" >
              <property name="text" >
               <string>Check 2</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" >
             <widget class="QPushButton" name="pushButton" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="text" >
               <string>Button 1</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1" >
             <widget class="QLineEdit" name="lineEdit_2" >
              <property name="text" >
               <string>Line Edit 2</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="QPushButton" name="pushButton_2" >
              <property name="text" >
               <string>Button 2</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1" >
             <widget class="QCheckBox" name="checkBox" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="text" >
               <string>Check 1</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" >
             <widget class="QSpinBox" name="spinBox" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="value" >
               <number>1</number>
              </property>
             </widget>
            </item>
            <item row="3" column="0" >
             <widget class="QRadioButton" name="radioButton" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="text" >
               <string>Radio 1</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0" >
             <widget class="QRadioButton" name="radioButton_2" >
              <property name="text" >
               <string>Radio 2</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" >
             <widget class="QSpinBox" name="spinBox_2" >
              <property name="value" >
               <number>2</number>
              </property>
             </widget>
            </item>
            <item row="2" column="0" >
             <widget class="QLineEdit" name="lineEdit" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="text" >
               <string>Line Edit 1</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0" >
             <widget class="QRadioButton" name="radioButton_3" >
              <property name="text" >
               <string>Radio 3</string>
              </property>
              <property name="checked" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="5" column="1" >
             <widget class="QCheckBox" name="checkBox_3" >
              <property name="text" >
               <string>Check 3</string>
              </property>
              <property name="checked" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QScrollBar" name="horizontalScrollBar" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBar_2" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="value" >
             <number>24</number>
            </property>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBar" >
            <property name="value" >
             <number>24</number>
            </property>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2" >
        <attribute name="title" >
         <string>Tab 2</string>
        </attribute>
        <widget class="QWidget" name="verticalLayout_2" >
         <property name="geometry" >
          <rect>
           <x>10</x>
           <y>10</y>
           <width>221</width>
           <height>261</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="leftMargin" >
           <number>0</number>
          </property>
          <property name="topMargin" >
           <number>0</number>
          </property>
          <property name="rightMargin" >
           <number>0</number>
          </property>
          <property name="bottomMargin" >
           <number>0</number>
          </property>
          <item>
           <layout class="QGridLayout" >
            <property name="leftMargin" >
             <number>0</number>
            </property>
            <property name="topMargin" >
             <number>0</number>
            </property>
            <property name="rightMargin" >
             <number>0</number>
            </property>
            <property name="bottomMargin" >
             <number>0</number>
            </property>
            <property name="horizontalSpacing" >
             <number>6</number>
            </property>
            <property name="verticalSpacing" >
             <number>6</number>
            </property>
            <item row="0" column="1" >
             <widget class="QComboBox" name="comboBox_2" >
              <item>
               <property name="text" >
                <string>Combo 2</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>Item 2</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>Item 3</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="0" >
             <widget class="QComboBox" name="comboBox" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <item>
               <property name="text" >
                <string>Combo 1</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="0" >
             <widget class="QComboBox" name="comboBox_3" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="editable" >
               <bool>true</bool>
              </property>
              <item>
               <property name="text" >
                <string>Combo 3</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="1" >
             <widget class="QComboBox" name="comboBox_4" >
              <property name="editable" >
               <bool>true</bool>
              </property>
              <item>
               <property name="text" >
                <string>Combo 4</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>Item 2</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>Item 3</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <property name="spacing" >
             <number>6</number>
            </property>
            <property name="leftMargin" >
             <number>0</number>
            </property>
            <property name="topMargin" >
             <number>0</number>
            </property>
            <property name="rightMargin" >
             <number>0</number>
            </property>
            <property name="bottomMargin" >
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label" >
              <property name="text" >
               <string>Label 1</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="horizontalSlider" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCalendarWidget" name="calendarWidget" />
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <includes/>
 <resources/>
 <connections/>
</ui>
